LDAR_Sim/model_code/input_mapper_v1.py

```diff
@@ -0,0 +1,102 @@
+# ------------------------------------------------------------------------------
+# Program:     The LDAR Simulator (LDAR-Sim)
+# File:        LDAR-Sim input mapper for version 1.0 files
+# Purpose:     Example input mapper
+#
+# Copyright (C) 2018-2020  Thomas Fox, Mozhou Gao, Thomas Barchyn, Chris Hugenholtz
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the MIT License as published
+# by the Free Software Foundation, version 3.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with this program.  If not, see <https://opensource.org/licenses/MIT>.
+#
+# ------------------------------------------------------------------------------
+
+def input_mapper_v1(parameters):
+    """Function to perform all input mapping from version 1.0 parameter files to presently compliant parameter files. This is
+    necessary to ensure reverse compatibility - to allow older parameter files to run properly.
```

Comment:
Do we just want to do mapping for V2 and beyond?

Reply:
Yeah I think so. This was left over from the second time I built this input mapper thing. It can be tossed - I've left it as an example discussion of all the different things to consider in input parameter mapping so we have a base to start all the work after version 2.0. As well, Soroush may wish to use this at some point depending on his needs, it could be easier for him to fill this in and turn it on than rebuild web app databases. I'm not sure.

Reply:
Yeah definitely leave in for now.

Reply:
Leaving for now.